RELEASE_NOTES.md

```diff
@@ -17,22 +17,52 @@ OPAE\'s goal is to accelerate FPGA adoption. It is a community effort to simplif
 More information about OPAE can be found
 at http://01.org/OPAE.
 
-Open Programmable Acceleration Engine (OPAE) 2.0.0 Release Notes
------------------------------------------------------------------
 
-This document provides the Release Notes for the Open Programmable
-Acceleration Engine (OPAE) 2.0.0 release.
-
-System Compatibility
---------------------
-
--   Hardware: tightly coupled FPGA products and programmable FPGA
-    acceleration cards for Intel(R) Xeon(R) processors:
-    - Intel(R) PAC with Arria(R) 10 GX FPGA (PCI ID: 0x09c4) FIM version 1.1.2-1 (1.2 Production)
-    - Intel® FPGA Programmable Acceleration Card N3000 (PCI ID: 0x0b30) FIM version 2.0.12 (2.0 Production)
-    - Intel® FPGA Programmable Acceleration Card D5005 (PCI ID: 0x0b2b) FIM version 2.0.12 (2.0 Production)
-    
--   Operating System: Tested on Fedora 31  with Linux Kernel 5.8 version.
+
+Open Programmable Acceleration Engine (OPAE) 2.0.0 Release Notes
+----------------------------------------------------------------------------------------------------------------------
+
+OPAE 2.0.0 release provides SDK, tools, and Linux kernel driver.  The main feature of this release is to support Intel®  FPGA Programmable Acceleration Card N3000 series.
+
+System Compatibility
+--------------------
+-   Hardware: Tightly coupled FPGA products and programmable FPGA  acceleration cards for Intel(R) Xeon(R) processors:
+- IntelÂ® FPGA Programmable Acceleration Card N3000-2 Production Release
+- IntelÂ® FPGA Programmable Acceleration Card N3000-V Production Release
+- IntelÂ® FPGA Programmable Acceleration Card N3000-3 Production Release
+
+
+Major Changes from 1.4.1 to 2.0.0
+---------------------------------
+- Added support to FPGA Linux kernel Device Feature List (DFL) driver.
+- Added support to PAC card N3000 series.
+- Added PACSign, bitstream_info, fpgasudpate, rsu, fpgaotsu, fpgaport  python tools.
+- Added ethernet tools for PAC card N3000.
+- Various bug fixes
+- Various memory leak fixes.

```

Comment:
I don't think we fixed memory leaks this time around.

Reply:
Removed memory leaks from Release notes